control/control.openSUSE.xml

```diff
@@ -449,6 +478,54 @@ is the most appropriate desktop for you.</label></desktop_dialog>
         <desktop_min_x><label>Minimal X Window</label></desktop_min_x>
         <desktop_enlightenment><label>Enlightenment Desktop</label></desktop_enlightenment>
 
+      <roles_caption>
+        <!-- TRANSLATORS: dialog caption -->
+        <label>Computer Role</label>
+        </roles_caption>
+        <roles_text>
+          <!-- TRANSLATORS: label in a dialog -->
+          <label>Computer Role are predefined use cases which tailor the system
+for the selected scenario.</label>
+        </roles_text>
+        <roles_help>
+          <!-- TRANSLATORS: dialog help -->
+          <label>&lt;p&gt;The computer role adjustments are in the range from package selection up
+to running additional services. By choosing a computer role, the system is
+configured accordingly to match the use case of the role. For Custom role additional dialog started which
```

Comment:
` For the &lt;em&gt;Custom&lt;/em&gt; role an additional dialog is displayed which allows manual software pattern selection.`